tripal_ws/api/tripal_ws.api.inc

```diff
@@ -255,22 +254,21 @@ function tripal_get_remote_content($site_id, $path = '', $query = '') {
   // If the data object has an error then this is some sort of
   // connection error (not a Tripal web servcies error).
   if (property_exists($data, 'error')) {
+    $error = '</pre>' . print_r($data['error'], TRUE) . '</pre>';
```

Comment:
I think this should be `$data->error`?

Reply:
Yep confirmed that changing $data['error'] to $data->error resolves all the issues I've encountered

Reply:
Whoops sorry about that.

Reply:
Can you commit your fix?

Reply:
I don't have access to the repo so I'd have to create a separate PR but @bradfordcondon does so I'll ask if he can push the change 👍

Reply:
I committed the change.